Add - between words in npm scripts

<!-- Please do your best to fill out all of the sections below! -->

## Description
I'm trying to submit my first PR to this awesome project.
There were some challenges that I was/am facing
e.g.
- I can't run `npm run format` from command line on Windows since it complains of not being able to run `.sh` scripts
C:\dev\idea\nx>npm run format
```bash
C:\dev\idea\nx>npm run format

> @nrwl/nx-source@10.1.0 format C:\dev\idea\nx
> ./scripts/format.sh

'.' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @nrwl/nx-source@10.1.0 format: `./scripts/format.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @nrwl/nx-source@10.1.0 format script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\vivzt\AppData\Roaming\npm-cache\_logs\2020-09-06T14_40_28_761Z-debug.log

```
- Running the same `format` script from my IDE (IntelliJ) results in `*.md` files changing to have `LF` line endings and show up as changed.
  - There are few ways we could resolve this issue. One option that comes to mind is adding `.gitattributes` to make this platform agnostic. I'm open to suggestions and willing to submit a PR for this based on Nx team's feedback.

----

Now about the trivial change I was trying to submit as my first PR - https://github.com/nrwl/nx/pull/3672
I'd like to suggest that the scripts 

- `checkcommit`
- `checkformat`
- `checkimports`
- `checkversions`

be renamed to:

- `check-commit`
- `check-format`
- `check-imports`
- `check-versions`

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -84,13 +84,13 @@ jobs:
           command: yarn documentation
       - run:
           name: Check Imports
-          command: yarn checkimports
+          command: yarn check-imports
       - run:
           name: Check Formatting
-          command: yarn checkformat
+          command: yarn check-format
       - run:
           name: Check Commit Message Format
-          command: yarn checkcommit
+          command: yarn check-commit
       - run:
           name: Run linting
           command: yarn lint

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -135,7 +135,7 @@ Please follow the following guidelines:
 - Make sure you run `yarn format`
 - Update documentation with `yarn documentation`. For documentation, check for spelling and grammatical errors.
 - Update your commit message to follow the guidelines below (use `yarn commit` to automate compliance)
-  - `yarn checkcommit` will check to make sure your commit messages are formatted correctly
+  - `yarn check-commit` will check to make sure your commit messages are formatted correctly
 
 #### Commit Message Guidelines
 

--- a/package.json
+++ b/package.json
@@ -8,16 +8,16 @@
   "scripts": {
     "build": "./scripts/package.sh --local",
     "commit": "git-cz",
-    "checkcommit": "node ./scripts/commit-lint.js",
+    "check-commit": "node ./scripts/commit-lint.js",
+    "check-format": "scripts/check-format.sh",
+    "check-imports": "node ./scripts/check-imports.js",
+    "check-versions": "ts-node -P ./scripts/tsconfig.scripts.json ./scripts/check-versions.ts",
     "e2e": "./scripts/e2e.sh",
     "e2e-ci": "./scripts/e2e-ci.sh",
     "format": "./scripts/format.sh",
     "nx-release": "./scripts/nx-release.js",
     "test": "nx run-many --target=test --all --parallel",
     "lint": "nx run-many --target=lint --all --parallel",
-    "checkformat": "scripts/check-format.sh",
-    "checkimports": "node ./scripts/check-imports.js",
-    "checkversions": "ts-node -P ./scripts/tsconfig.scripts.json ./scripts/check-versions.ts",
     "depcheck": "ts-node -P ./scripts/tsconfig.scripts.json ./scripts/depcheck.ts",
     "local-registry": "./scripts/local-registry.sh",
     "documentation": "./scripts/documentation/documentation.sh && ./scripts/documentation/check-documentation.sh",
@@ -251,7 +251,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "yarn checkcommit && yarn documentation && pretty-quick --check"
+      "pre-push": "yarn check-commit && yarn documentation && pretty-quick --check"
     }
   }
 }

--- a/scripts/check-versions.ts
+++ b/scripts/check-versions.ts
@@ -3,13 +3,13 @@
  * It uses naming conventions to transform constants to matching node module name.
  *
  * Usage:
- *   yarn checkversions [file]
+ *   yarn check-versions [file]
  *
  * Positional arg:
  *   - [file]: relative or absolute file path to the versions file.
  *
  * Example:
- *   yarn checkversions packages/react/src/utils/versions
+ *   yarn check-versions packages/react/src/utils/versions
  */
 
 import { join } from 'path';

